src/components/TabBar.tsx

```diff
@@ -101,6 +102,7 @@ export const Tab = styled.div<{ isActive: boolean }>`
         `
       : css`
           color: ${theme.font.tertiary};
+          background-color: ${({ theme }) => colord(theme.bg.background2).alpha(0.7).toHex()};
```

Comment:
What's that for?

Reply:
So the left and right sides of the tabs were not the color of the page background because of where this background prop was. Moving it here maintains the effect that we want in the inactive element without changing the bg color of the whole row.